contributing/code/bc.rst

```diff
@@ -258,6 +258,14 @@ Make public or protected                                                  Yes
 Remove private property                                                   Yes
 **Constructors**
 Add constructor without mandatory arguments                               Yes             :ref:`[1] <note-1>`
+:ref:`Add argument without a default value <add-argument-public-method>`  No
+Add argument with a default value                                         Yes
```

Comment:
We should add a footnote that this is only allowed at the end of the parameter list (similar to `[3]`, but then replacing removing with adding).

Reply:
New footnote added, does it look good to you?